components/camel-splunk-hec/src/main/java/org/apache/camel/component/splunkhec/SplunkHECConfiguration.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.splunkhec;
+
+import java.net.UnknownHostException;
+
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriParams;
+import org.apache.camel.util.HostUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@UriParams
+public class SplunkHECConfiguration {
+    private static final transient Logger LOG = LoggerFactory.getLogger(SplunkHECConfiguration.class);
+
+    @UriParam(label = "producer")
+    private String index = "camel";
+    @UriParam(label = "producer")
+    private String sourceType = "camel";
+    @UriParam(label = "producer")
+    private String source = "camel";
+    @UriParam(label = "producer")
+    private String host;
+    @UriParam(label = "producer")
+    private boolean skipTlsVerify;
+    @UriParam(label = "producer")
```

Comment:
You need to specify default value in @UriParam when it has a value, also for the ones that has camel above.